data-serving/scripts/import-data/src/index.ts

```diff
@@ -1 +1,71 @@
-new Mongo()
\ No newline at end of file
+declare let args: SetupDatabaseParameters;
+
+interface SetupDatabaseParameters {
+    databaseName: string;
+    collectionName: string;
+    schemaPath: string;
+}
+
+const setupDatabase = async ({
+    databaseName,
+    collectionName,
+    schemaPath,
+}: SetupDatabaseParameters): Promise<void> => {
+    try {
+        const schema = JSON.parse(await cat(schemaPath));
+        print(`Read schema from ${schemaPath}`);
+
+        // Connect to the default MongoDb instance.
+        const connection: Connection = new Mongo();
+        print(`Connected to instance at ${connection['host']}`);
+
+        // Get or create the database. (If it doesn't exist, it will be created.)
+        const database = await connection.getDB(databaseName);
+        print(`Connected to database "${database}"`);
+
+        // Either update the collection with the schema if it already exists, or
+        // create the collection with the schema.
+        let collection;
+        if (
+            (await database.getCollectionNames()).some(
+                (c) => c == collectionName,
+            )
+        ) {
+            // Drop existing data, if any.
+            collection = await database.getCollection(collectionName);
+            collection.remove({});
```

Comment:
FYI I had a look at dropDatabase (https://docs.mongodb.com/manual/reference/method/db.dropDatabase/) but it seems that we need to restart a bunch of things if we create a new database after doing that, so doing it the way you did looks good.

Reply:
SG, thanks!